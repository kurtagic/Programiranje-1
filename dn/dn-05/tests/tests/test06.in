61
11 64
11 84
11 89
11 85
11 72
11 85
11 40
11 92
11 86
11 76
11 60
11 93
11 82
11 70
11 84
11 59
11 53
11 84
11 60
11 68
11 99
11 66
11 39
11 34
11 65
11 50
11 79
11 66
11 33
11 78
11 42
11 39
11 37
11 66
11 92
11 76
11 48
11 76
11 71
11 56
11 32
11 70
11 79
11 55
11 97
11 38
11 38
11 74
11 52
11 77
11 100
11 89
11 82
11 75
11 90
11 75
11 74
11 57
11 59
11 98
11 37
