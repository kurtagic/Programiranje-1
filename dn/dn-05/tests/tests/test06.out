32: 3
33: 5
34: 4
35: 3
37: 10
38: 12
39: 11
40: 7
41: 3
42: 1
43: 3
48: 1
49: 3
50: 1
51: 3
52: 2
53: 4
54: 3
55: 2
56: 4
57: 4
58: 6
59: 6
60: 8
61: 7
64: 1
65: 4
66: 10
67: 12
68: 1
69: 3
70: 5
71: 7
72: 6
73: 3
74: 12
75: 14
76: 13
77: 10
78: 12
79: 5
80: 7
82: 4
83: 6
84: 11
85: 13
86: 10
87: 12
89: 4
90: 7
91: 9
92: 5
93: 7
94: 6
97: 1
98: 4
99: 6
100: 4
101: 6
